ParticlesKit/ParticlesKit/_Credentials/JsonCredentialsProvider.swift

```diff
@@ -17,7 +17,7 @@ open class JsonCredentialsProvider: NSObject {
 
     private var entity: DictionaryEntity?
 
-    public func key(for lookupKey: String) -> String? {
+    public func credential(for lookupKey: String) -> String? {
```

Comment:
renamed to `credential` since `key` is confusing to me. The credential is not always a key